### DISCUSSION TITLE
npm install from verdaccio is launching postinstall when it should not

### DISCUSSION THREAD

**Discussion by lmdc45** [verdaccio/verdaccio#2259] [Q&A] [answered]
This is about this issue:

#https://github.com/maxmind/GeoIP2-node/issues/377#issuecomment-841621445

When I install using 
`npm i @maxmind/geoip2-node --registry=https://registry.npmjs.org`
all is working fine

However when I install through verdaccio, the postinstall script is launched (and breaks).

It seems the package.json file holding all info for all version on the verdaccio storage is different from the one in the tarball (script name is written "_postinstall" in tarball storage however written "postinstall" in verdaccio storage)

Just wondering if anything is wrong or if this is normal behaviour, and if I am missing an option or argument somewhere

**Comment by juanpicado** [comment, marked as answer]
Verdaccio does not execute code in the client side.
